internals/cli/last.go

```diff
@@ -33,9 +33,9 @@ var changeIDMixinOptDesc = map[string]string{
 	"last": "Select last change of given type (install, refresh, remove, try, auto-refresh, etc.). A question mark at the end of the type means to do nothing (instead of returning an error) if no change of the given type is found. Note the question mark could need protecting from the shell.",
 }
 
-var changeIDMixinArgDesc = []argDesc{{
-	name: "<change-id>",
-	desc: "Change ID",
+var changeIDMixinArgDesc = []PositionalArg{{
```

Comment:
If you do plan to change the struct name on the right, would it make sense to change the variable name on the left, in the same way you've aligned the names for ```OptionsHelp``` variables?

Reply:
Definitely, but as I knew this would be a set of controversial changes, I wanted to come to an agreement on naming the structs first before I went ahead and created a bunch of noise by renaming everything under the sun (: